### PR TITLE
Updated the errors to wrap comments

### PR DESCRIPTION
**Pull request by snkzt**

### CODE REVIEW

File: cmd/speep/main.go [hunk 1]
```diff
@@ -73,6 +73,7 @@ func HandleGet(getCmd *flag.FlagSet, all *bool, keyword *string) error {
 		// Return full shortcut list
 		shortcuts, err := shortcuts.GetShortcuts()
 		if err != nil {
+			err = fmt.Errorf("Failed to acquire the existing list: %w", err)
 			return err
```
Comment (kinbiko) [inline cmd/speep/main.go]: Whenever you have an assignment before a return you can just return the stuff on the right hand side of the equals instead
```suggestion
			return fmt.Errorf("Failed to acquire the existing list: %w", err)
```

This comment applies throughout the PR.

File: cmd/speep/main.go [hunk 2]
```diff
@@ -84,6 +85,7 @@ func HandleGet(getCmd *flag.FlagSet, all *bool, keyword *string) error {
 	if *keyword != "" {
 		shortcuts, err := shortcuts.GetShortcuts()
 		if err != nil {
+			err = fmt.Errorf("Failed to acquire the existing list: %w", err)
```
Comment (kinbiko) [inline cmd/speep/main.go]: Errors should not be capitalized unless it's a proper noun or an acronym

**Comment by snkzt** [comment]
@kinbiko
I updated the path assignment for `$XDG_CONFIG_HOME` as in [here](https://github.com/snkzt/shortcut-peeper/pull/4/commits/6c8b3ddbfbdfd320aa296dcf65016054aca8dbc0#diff-43434f011f9b006c496e06d86e9ae1930b8e6c6b44de090007d9203696228e36R15-R25) but when run a command `speep get -all` to check the change, still get the old error as below.
```
open $XDG_CONFIG_HOME/shortcuts.json: no such file or directory
```

I run build the app before run the command to apply the change and compile, but still not working and not sure why.
Do you see any problem in the process?

**Comment by kinbiko** [comment]
I get a "better" error message than that, when I try and use this branch:

```
failed to acquire the existing list: no list exists: open /Users/kinbiko/.config/speep/shortcuts.json: no such file or directory
```

So maybe you're not running the latest code, but something old instead?

I guess in this case (before the user has created a shortcut to look up) we should return a better error message.

**Comment by snkzt** [comment]
Thank you!
> I guess in this case (before the user has created a shortcut to look up) we should return a better error message.

Indeed. I will update the error message.

> So maybe you're not running the latest code, but something old instead?

Likely. I will try delete a binary file in the local path and try install again.

File: cmd/speep/main.go (hunk 1 above)
Comment (snkzt) [reply cmd/speep/main.go]: Thank you all addressed!

File: cmd/speep/main.go (hunk 2 above)
Comment (snkzt) [reply cmd/speep/main.go]: Thank you all fixed!

**Pull request merged**